data/product.d/rhvh.conf

```diff
@@ -26,6 +36,8 @@ req_partition_sizes =
 
 [User Interface]
 help_directory = /usr/share/anaconda/help/rhv
+hidden_spokes =
```

Comment:
The tests are failing, because `rhvh.conf` and `ovirt.conf` are expected to have the same configuration. Could you please define `hidden_spokes` also in `ovirt.conf`?

Reply:
Fixed.